### ISSUE TITLE
SafeAreaUtils.kt Unresolved reference: R

### ISSUE THREAD

**Issue by alirehmanappcrates** [th3rdwave/react-native-safe-area-context#292]
I'm facing an issue related to SafeAreaUtils.kt while trying to run the app on the emulator. **Unresolved reference: R**
Image attached

<img width="350" alt="Screenshot 2022-06-20 at 6 23 58 PM" src="https://user-images.githubusercontent.com/86340695/174611932-186d85cd-5632-46f8-94f3-63c43a71d1ea.png">

```
System:
    OS: macOS 12.4
    CPU: (8) x64 Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz
    Memory: 131.58 MB / 8.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.13.1 - ~/.nvm/versions/node/v16.13.1/bin/node
    Yarn: 1.22.18 - ~/.nvm/versions/node/v16.13.1/bin/yarn
    npm: 8.1.2 - ~/.nvm/versions/node/v16.13.1/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK:
      API Levels: 26, 29, 30, 31
      Build Tools: 28.0.3, 29.0.2, 30.0.2, 31.0.0, 32.0.0
      System Images: android-29 | Intel x86 Atom_64, android-29 | Google APIs Intel x86 Atom, android-30 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8609683
    Xcode: 13.4/13F17a - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.11 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.1 => 17.0.1 
    react-native: 0.64.2 => 0.64.2 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

**Comment by HaddyYang** [comment]
I have some issue

**Comment by Hassan-Riaz** [comment]
@alirehmanappcrates  @HaddyYang  I was facing the exact same issue.
Fixed it by downgrading to 3.2.0

`npm i react-native-safe-area-context@3.2.0`

`yarn add react-native-safe-area-context@3.2.0`

**Comment by jacobp100** [comment]
Is this still an issue - or can I close this ticket?
